backend/controllers/commandControllers.js

```diff
@@ -0,0 +1,128 @@
+const asyncHandler = require("express-async-handler");
+const dotenv = require("dotenv");
+// const { OpenAI } = require('openai');
+const Message = require("../models/messageModel");
+const Chat = require("../models/chatModel");
+const User = require("../models/userModel");
+const { GoogleGenerativeAI } = require("@google/generative-ai");
+
+dotenv.config()
+
+// const baseURL = "https://api.aimlapi.com/v1";
```

Comment:
can we separate this to a new file

do like this 
integrations
 - chatgpt
 - gemini

Reply:
Ok , sure